src/main/java/org/folio/circulation/infrastructure/storage/sessions/PatronActionSessionRepository.java

```diff
@@ -230,11 +231,13 @@ private boolean isPatronActionTypeSpecified(PatronActionType actionType) {
       .thenApply(mapResult(loans -> setLoansForSessionRecords(sessionRecords, loans)));
```

Comment:
Why do we fetch campuses and institutions, but not libraries? Shouldn't they always go together?